docs/history.md

```diff
@@ -34,4 +34,12 @@ move the repository, so we simply forked it. We still welcome the HackMD team
 as part of our community, especially since a large portion of this code base
 originated with them.
 
-*For the debate that lead to this step, please refer to the [governance debate](https://github.com/hackmdio/hackmd/issues/1170) and [the announcement of the new repository](https://github.com/codimd/server/issues/10).*
+*For the debate that lead to this step, please refer to the [governance debate](https://github.com/hackmdio/hackmd/issues/1170) and [the announcement of the new repository](https://github.com/hedgedoc/hedgedoc/issues/10).*
+
+## CodiMD became HedgeDoc
+
+Many people opened issues in the codimd server repository for bugs and features, which they saw at the HackMD codimd.
+The fact, that there have been two projects with the same purpose and the same name was very confusing.
+The solution was to find a unique, new name. This process started in April 2020 with a [call to participation](https://community.codimd.org/t/renaming-yet-another-time/102).
+It took some time, but in [July 2020 the community agreed on the name "HedgeDoc"](https://community.codimd.org/t/codimd-becomes-hedgedoc/170) and some months later [a new logo was found](https://community.codimd.org/t/time-to-find-the-hedgedoc-logo/171/4).
+Finally, in November 2020 version 1.7 was released. Even if the renaming was originally planned for version 2.0, the dev team decided to release 1.7 under the new name to create a smooth transition.
```

Comment:
Here I would replace dev with the full word development. I'm also not sure if the original plan vs actual plan is that important here. It might be more useful to tell about the 2.0 development that happened in parallel and the unexpected release of 1.7.0 even when the release notes of 1.6.0 stated something else.